src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js

```diff
@@ -244,17 +249,29 @@ export default {
       if (columnName === 'LineDescription') {
         return row.lineDescription
       }
-      const currency = this.pointOfSalesCurrency.iSOCode
+      const currencyCode = this.pointOfSalesCurrency.iSOCode
       if (columnName === 'CurrentPrice') {
-        return this.formatPrice(row.priceActual, currency)
+        return this.formatPrice({
+          value: row.priceActual,
+          currencyCode
+        })
       } else if (columnName === 'QtyOrdered') {
-        return this.formatQuantity(row.quantityOrdered)
+        return this.formatQuantity({
+          value: row.quantityOrdered
+        })
       } else if (columnName === 'Discount') {
         return this.formatPercent(row.discount / 100)
       } else if (columnName === 'GrandTotal') {
-        return this.formatPrice(row.grandTotal, currency)
+        return this.formatPrice({
+          value: row.grandTotal,
+          currencyCode
+        })
       } else if (columnName === 'ConvertedAmount') {
-        return this.formatPrice(this.getTotalAmount(row.grandTotal, this.totalAmountConvertedLine.multiplyRate), this.totalAmountConvertedLine.iSOCode)
+        const value = this.getTotalAmount(row.grandTotal, this.totalAmountConvertedLine.multiplyRate)
```

Comment:
Just can see change for source code with the same functionality, please let it without changes